bokehjs/src/coffee/models/glyphs/webgl/line.coffee

```diff
@@ -692,6 +692,7 @@ class LineGLGlyph extends BaseGLGlyph
       @prog.set_uniform('u_scale_aspect', 'vec2', [sx, sy])
       @prog.set_uniform('u_scale_length', 'float', [scale_length])
 
+      @I_triangles = mainGlGlyph.I_triangles
```

Comment:
Right. because lines can only be selected as a whole, i.e. the `indices` arg is always the complete line.